qiime2/sdk/util.py

```diff
@@ -188,7 +188,7 @@ def get_available_usage_drivers() -> Dict[str, 'UsageDriver']:
     }
 
 
-@python_app
+@python_app(executors=['default'])
```

Comment:
Should this actually be the executor of the current app?

Reply:
I was wondering about that. I don't feel like it really matters? I only set it to default here to make it at least concretely defined.